vkubelet/vkubelet.go

```diff
@@ -117,12 +139,18 @@ func New(ctx context.Context, cfg Config) (s *Server, retErr error) {
 		}
 	}()
 
-	return s, nil
-}
+	go func() {
+		tick := time.Tick(time.Minute)
+		for {
+			select {
+			case <-ctx.Done():
+				return
```

Comment:
Can you please confirm the program exists when this goroutine returns?

Sorry if I'm wasting your time, but I've been bitten before by `time.Tick` leaking the underlying ticker even if a goroutine ends. If the program exits when this goroutine exits, it's fine.

Reply:
You are right, we shouldn't assume the application is exiting in this case. There are some other tickers to update as well.